### ISSUE TITLE
Instructions for Browserify?

### ISSUE THREAD

**Issue by silouanwright** [tcoopman/ga-react-router#3]
I'm not really familiar with Webpack, so it would be nice to know how to use it with Browserify.

**Comment by tcoopman** [comment]
I don't use Browserify myself, but I believe it should work with https://www.npmjs.com/package/browserify-replace

`browserify -t [ browserify-replace --replace '{ "from": "GA_TRACKING_CODE", "to": "XXXXXXXX" }' -o app.browser.js app.js`

Does this help you? If it does, I'll add it to the README

**Comment by mindjuice** [comment]
I am using `ga-react-router` with an older Mimosa-based project.  I also needed to be able to change the `GA_TRACKING_CODE` at runtime so I can use the same code on my staging and production servers.

I just did the following in my main React component:

```
window.GA_TRACKING_CODE = isProd ? "UA-XXXXX111" : "UA-XXXXX222";
```

That will work regardless of what build tool you're using, although it's not isomorphic.

If you want to render under node on your server, you will have to do something like this: http://stackoverflow.com/questions/4224606/how-to-check-whether-a-script-is-running-under-node-js
